### ISSUE TITLE
Misleading instructions

### ISSUE THREAD

**Issue by sarah-dev-av** [decalage2/oletools#547]
In the documentation https://github.com/decalage2/oletools/wiki/olevba, it says "MS Office files encrypted with a password are also supported, because VBA macro code is never encrypted, only the content of the document". This makes me think, for an encrypted office file, olevba can extract the macro with or without a password provided. However, this is not the case. olevba seems only extracts the macro when it gets the right password either from user or the default list.

**Comment by decalage2** [comment]
Good catch, I should update the documentation: This statement is only true for OLE files (Office 97-2003 old formats). For the new formats, encryption includes the VBA code, so you need to provide the password to decrypt.
